packages/installer.vm/tools/chocolateyinstall.ps1

```diff
@@ -32,6 +32,22 @@ try {
     }
     VM-Write-Log "INFO" "Packages installation complete"
 
+    # Create an Admin Command Prompt shortcut to pin to taskbar
+    $toolName = 'Admin Command Prompt'
+    $category = 'Utilities'
```

Comment:
I am not sure about installing this inside a category, see my comment in https://github.com/mandiant/flare-vm/pull/531/files#r1400442141

Reply:
Moved it over to `Env{RAW_TOOLS_DIR}`